### ISSUE TITLE
proguard rules for testBuildType seemingly ignored

### ISSUE THREAD

**Issue by rhodey** [stanfy/spoon-gradle-plugin#59]
in our [build.gradle](https://github.com/WhisperSystems/TextSecure/blob/master/build.gradle) we specify a custom build type using `testBuildType`, this build type requires proguard because we want to test an APK as similar to our release build as possible. when I run `./gradlew clean spoon` proguard complains **a lot** and all about things that have already been solved within our proguard rules.

in short, changes to the proguard rules of our build type specified by `testBuildType` seem to have no effect when including `apply plugin: 'spoon'` in our `build.gradle` file.

**Comment by roman-mazur** [comment]
And do you see this problem if you run tests with `connectedCheck`?

**Comment by roman-mazur** [comment]
Overall, this plugin does not affect how test apk is built. We rely on Android plugin output apk and then simply pass it to spoon to install on a device and test.

Feel free to reopen if you have any new data or comments.
